extensions/typescript-language-features/src/tsServer/webServer.ts

```diff
@@ -0,0 +1,539 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+/// <reference types='typescript/lib/tsserverlibrary' />
```

Comment:
Could the tsserver cod ebe in its own `tsconfig` project? (and own directory under`typescript-language-features`?